gobcore/utils.py

```diff
@@ -130,5 +130,5 @@ def get_filename(name, offload_folder):
     dir = os.path.join(GOB_SHARED_DIR, offload_folder)
```

Comment:
path = Path(GOB_SHARED_DIR, offload_folder)
path.mkdir(exist=True, parents=True)
return str(path)

Mix path and os.path.

Reply:
Also if the function is called get_filename but it also creates a parent dir, maybe give it a better name, or split functionality?

Where this is used:
bla = self.get_filename()  # return type Path instead
bla.mkdir(exist_ok=True, parents=True)